pkg/nordic_softdevice_ble/src/ble-mac.h

```diff
@@ -0,0 +1,50 @@
+#ifndef BLE_MAC_H
+#define BLE_MAC_H
+
+typedef enum {
+    BLE_EVENT_RX_DONE = 20000,
+} ble_mac_event_enum_t;
+
+#define BLE_IFACE_ADDED             (10000)
+
+#define BLE_SIXLOWPAN_MTU           (1280U)
+#define BLE_SIXLOWPAN_L2_ADDR_LEN   (8)
```

Comment:
The current mainline linux btle 6lowpan is broken in many ways.

@kaspar030 I cc'ed you in a patch series which should fix everything. This patch series isn't mainline now but can be applied on bluetooth-next kernel repository. There exists some bug in L2, I explained everything in the patch-series.

I will bring two btle dongles to riot-summit and can demonstrate some linux <-> linux communication (with the patch series which should fix everything) and some possible future ideas. etc.